package.json

```diff
@@ -13,8 +13,10 @@
     "lib"
   ],
   "scripts": {
-    "test": "ts-node node_modules/jasmine/bin/jasmine.js",
+    "test": "npm run prettier && ts-node node_modules/jasmine/bin/jasmine.js",
```

Comment:
I wouldn't automatically run it on `npm test`, in this case doesn't matter much, but in general it would unnecessarily slow down test.

I would have `npm format` that formats and `npm format:check` that runs prettier with `--check`, then we can have check as part of CI.